Makefile

```diff
@@ -1,85 +1,98 @@
-arch = x86_64
-version = 39
-base_dir = $(shell pwd)
-image_repo = ghcr.io/ublue-os
-image_name = base-main
-image_tag = $(version)
-variant = Silverblue
-web_ui = false
-
-image_repo_escaped = $(subst /,\/,$(image_repo))
-image_repo_double_escaped = $(subst \,\\\,$(image_repo_escaped))
-
-ifeq ($(variant),'Server')
-lorax_args = --macboot --noupgrade
+# Configuration vars
+## Formatting = UPPERCASE
+ARCH = x86_64
+VERSION = 39
+IMAGE_REPO = ghcr.io/ublue-os
+IMAGE_NAME = base-main
+IMAGE_TAG = $(version)
+VARIANT = Silverblue
```

Comment:
This should match the workflow defaults

Reply:
While I don't think this must match the workflow, it is correct that "Silverblue" variant is the wrong one to use for `base-main`.

Reply:
Yup. Agree after reading what was said earlier